php/public/index.php

```diff
@@ -140,7 +142,7 @@
 });
 
 // Auth Redirector
-$app->get('/', function (\Psr\Http\Message\RequestInterface $request, \Psr\Http\Message\ResponseInterface $response, $args) use ($container) {
+$app->get('/', function (\Psr\Http\Message\RequestInterface $request, Response $response, mixed $args) use ($container) {
```

Comment:
```suggestion
$app->get('/', function (Request $request, Response $response, mixed $args) use ($container) {
```

consistency

Reply:
No, RequestInterface should be correct IIRC... (It is not a ServerRequest but a Client-Side request IIRC)...

Reply:
Name is misleading then, maybe import both as Request and ServerRequest to avoid the confusion.

Reply:
sorry, saw this too late